tests/translate/translate_divergencedamping.py

```diff
@@ -30,5 +30,9 @@ def __init__(self, grid):
         }
         self.max_error = 3.0e-11
 
-    def compute(self, inputs):
-        return self.column_split_compute(inputs, {"nord": "nord_col", "d2_bg": "d2_bg"})
+    def compute_from_storage(self, inputs):
+        divdamp = DivergenceDamping(spec.namelist, inputs["nord_col"], inputs["d2_bg"])
```

Comment:
because we need the values from nord and d2_bg to initialize, which we are getting here from the serialized data. We do know though how these are computed, so it is possible to get them from calling d_sw.get_column_namelist. I am open to that approach. There are other tests that initialize in the translate's compute method -- actually Fillz has been updated to take serialized data in init and in master is now initialized in the compute method:
` run_fillz = fillz.FillNegativeTracerValues(
            inputs.pop("im"), inputs.pop("jm"), inputs.pop("km"), inputs.pop("nq")
        )`

Reply:
if we do go the route of computing the nord and d2_bg columns rather than reading them from serialized data, we need to introduce dependencies on the model code, e.g. d_sw's get-column_namelist, and then need the currently globally available namelist... These two things could change requiring more work on this class, while if we read from the data, it can survive a little longer without modification. Of course you could argue we need to be testing with the namelist we are defining in code to catch mistakes in that definition, but hopefully the D_SW test would catch that. I think I'll leave it for now, but am open to discussing more and changing in the next divergence damping PR if that's ok.